documentation/book/con-kafka-broker-external-listeners-ingress.adoc

```diff
@@ -68,7 +68,7 @@ listeners:
 On `ingress` listeners, you can use the `dnsAnnotations` property to add additional annotations to the ingress resources.
 You can use these annotations to instrument DNS tooling such as {KubernetesExternalDNS}, which automatically assigns DNS names to the ingress resources.
 
-.Example of an external listener of type `ingress` using {KubernetesExternalDNS} annotations
+.Example of an external listener of type `ingress` using Kubernetes External DNS annotations
```

Comment:
This is a project named External DNS. I'm fine with not having the link here. But I'm not sure _Kuberneets External DNS_ is the right name - maybe just _External DNS_. Also, can we somehow typografically distinguish it here to make it more clear this is a name and not _whatever external DNS_?

Reply:
Gargh! I think we can live without a version number. :thumbsup: